code/modules/lighting/light_effect_cast.dm

```diff
@@ -66,14 +66,16 @@ var/light_post_processing = 1 // Use writeglobal to change this
 		alpha = initial(alpha)
 		animate(src, alpha = initial(alpha) - rand(30, 60), time = 2, loop = -1, easing = SINE_EASING)
 
-	for (var/mob/M in view(world.view, src))
-		M.check_dark_vision()
-
-	for(var/turf/T in view(2*light_range, src))
-		T.lumcount = -1
-		affecting_turfs += T
-		if (get_dist(T, location) <= distance_to_wall_illum && CHECK_OCCLUSION(T))
-			affected_shadow_walls += T
+	for (var/thing in view(world.view, src))
```

Comment:
since most light objs have a light_range of 5, this effectively divide the numbers of turfs you iterate on by 50%.